geth/jail/jail.go

```diff
@@ -137,47 +140,68 @@ func (j *Jail) initCell(cell *Cell) error {
 }
 
 // CreateAndInitCell creates and initializes a new Cell.
-func (j *Jail) createAndInitCell(chatID string, code ...string) (*Cell, error) {
+func (j *Jail) createAndInitCell(chatID string, code ...string) (*Cell, string, error) {
 	cell, err := j.obtainCell(chatID, false)
 	if err != nil {
-		return nil, err
+		return nil, "", err
 	}
 
 	if err := j.initCell(cell); err != nil {
-		return nil, err
+		return nil, "", err
 	}
 
-	// Run custom user code
+	resultString, err := j.runJSCode(cell, code)
+	if err != nil {
+		return nil, "", err
+	}
+
+	return cell, resultString, nil
+}
+
+// runJSCode runs the provided code in the provided cell and
+// returns results of the last statement.
+func (j *Jail) runJSCode(cell *Cell, code []string) (string, error) {
```

Comment:
This function looks like a good candidate to be an internal method of `Cell` type, rather than `Jail`.

Reply:
Actually, I figured out that I can use `cell.Run` for that (and that is used in the method just above that).